lib/webengine.h

```diff
@@ -16,8 +16,19 @@
 #include <QString>
 #include <qmozcontext.h>
 
+/*!
+  \module SailfishWebView
+*/
+
+
 namespace SailfishOS {
 
+/*!
```

Comment:
Qt style prefers docs in .cpp rather than .h to avoid cluttering .h

Reply:
Thanks, I've shifted this to the cpp. As an aside, I've always found it a bit strange that the docs are in the implementation rather than the header file.